docs/source/programming.rst

```diff
@@ -828,8 +828,9 @@ sqrt (w)
     Supports unsigned integer (result is ubyte) and floating point numbers.
     To do the reverse - squaring a number - just write ``x*x``.
 
-divmod (number, divident, division, remainder)
-    Performs division and remainder calculation in a single call. This is faster than using separate '/' and '%' calculations.
+divmod (dividend, divisor, quotient, remainder)
```

Comment:
builtinfunctions.kt also contains these names for the parameters, but I can change those .